frontend/src/pages/LogsPage/LogsTable/LogsTable.tsx

```diff
@@ -108,7 +108,7 @@ export const LogsTable = ({ data, loading, query }: Props) => {
 	useEffect(() => {
 		// Collapse all rows when search changes
 		table.toggleAllRowsExpanded(false)
-	}, [logEdges])
+	}, [logEdges, table])
```

Comment:
See https://github.com/highlight/highlight/pull/4393#discussion_r1119002239

Prior to this change, we had an infinite loop:

![Screenshot 2023-02-27 at 4 43 45 PM](https://user-images.githubusercontent.com/58678/221744128-4f50db51-8f73-4e95-b35f-c62bc5a76236.png)

After this change, the error goes away.